db_lesson.sql

```diff
@@ -0,0 +1,467 @@
+db_lesson_work2
+
+
+MariaDB [(none)]> show databases;
++--------------------+
+| Database           |
++--------------------+
+| db_lesson          |
+| information_schema |
+| mysql              |
+| performance_schema |
+| phpmyadmin         |
+| test               |
++--------------------+
+6 rows in set (0.001 sec)
+
+MariaDB [(none)]> use db_lesson;
+Database changed
+MariaDB [db_lesson]> show tables;
++---------------------+
+| Tables_in_db_lesson |
++---------------------+
+| people              |
+| reports             |
++---------------------+
+2 rows in set (0.001 sec)
+
+
+
+-- Q1------------------------------------------------------------------------------------------------
+-- departmentsテーブルの追加
+
+MariaDB [db_lesson]> create table departments (
+    -> department_id int unsigned auto_increment primary key,
+    -> name varchar(20) not null,
+    -> created_at timestamp null default current_timestamp,
+    -> updated_at timestamp null default current_timestamp on update current_timestamp
+    -> );
```

Comment:
- `-> `　←こちらもSQL文ではなく改行を表す表示なので、`MariaDB [db_lesson]> `とともに消しておきましょう！

```suggestion
create table departments (
department_id int unsigned auto_increment primary key,
name varchar(20) not null,
created_at timestamp null default current_timestamp,
updated_at timestamp null default current_timestamp on update current_timestamp
);
```

Reply:
MariaDB [db_lesson]>と改行を削除しました。